test_graphs.py

```diff
@@ -2,18 +2,22 @@
 import pytest
 
 from run_config_input import run_normal_config_file
-from utils.random_graph_data import random_graph_data
+from utils.deterministic_graph_data import deterministic_graph_data
 from utils.utils import get_comm_size_and_rank
 from test_trained_model import test_trained_model
 
+import torch
+
+torch.manual_seed(0)
```

Comment:
Why is this here?

Reply:
I thought that fixing the random seed would help ensure that the unit_tests run somewhat consistently on different machines. Do you recommend thatI remove it?

Reply:
I just wasn't familiar with the function and was confused why it wasn't inside `pytest_train_model`